### PR TITLE
include useful uitype and units info in sample xml snippets

### PR DESCRIPTION
**Pull request by rwcarlsen**
snapshot1

### CODE REVIEW

**Comment by gidden** [comment]
looks good, thanks @rwcarlsen

File: source/cyclusagent.py [hunk 1]
```diff
@@ -162,13 +173,18 @@ def buildsample(cpptype, schematype=None, uitype=None, names=None):
         name = 'val'
         if names is not None:
             name = names
-        d_type = _type(t, schematype or uitype)
-        impl += '<{0}>[{1}]</{0}>'.format(name, d_type)
+        d_type =  _type(t, schematype or uitype)
```
Comment (gidden) [inline source/cyclusagent.py]: two spaces here

**Comment by gidden** [comment]
small style edit

File: source/cyclusagent.py (hunk 1 above)
Comment (rwcarlsen) [inline source/cyclusagent.py]: fixed

**Comment by gidden** [comment]
separate question, do we have a place where we explain what valid `nuclide`s are?

**Comment by rwcarlsen** [comment]
I don't think so - we should probably add one:

    92235, 922350000, u235, U235, u-235, U-235

**Comment by FlanFlanagan** [comment]
Hey, pigging backing on this. What would you like nuclide to show in the
ui. I don't have behavior for it now

**Comment by mbmcgarry** [comment]
I vote for one of the letter-number combos


On May 19, 2015, at 10:00 PM, FlanFlanagan <notifications@github.com<mailto:notifications@github.com>> wrote:

Hey, pigging backing on this. What would you like nuclide to show in the
ui. I don't have behavior for it now
On May 19, 2015 9:48 AM, "Robert Carlsen" <notifications@github.com<mailto:notifications@github.com>> wrote:

> I don't think so - we should probably add one:
>
> 92235, 922350000, u235, U235, u-235, U-235
>
> On Tue, May 19, 2015 at 8:43 AM, Matthew Gidden <notifications@github.com<mailto:notifications@github.com>>
> wrote:
>
> > Merged #146 <https://github.com/cyclus/cyclus.github.com/pull/146>.
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <https://github.com/cyclus/cyclus.github.com/pull/146#event-308530384>.
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cyclus/cyclus.github.com/pull/146#issuecomment-103532437>
> .
>

—
Reply to this email directly or view it on GitHub<https://github.com/cyclus/cyclus.github.com/pull/146#issuecomment-103736114>.

*************************************************************************
Meghan McGarry, PhD ~ http://mbmcgarry.github.io/
Postdoctoral Fellow, Consortium for Verification Technology
Computational Nuclear Engineering ~ http://cnerg.engr.wisc.edu
Engineering Physics ~ UW-Madison ~ 608-263-6974
*************************************************************************

**Comment by rwcarlsen** [comment]
Radio - I think a text box is fine, and just let the user put it in
whatever format they want.  The cyclus input file can handle all of the
formats I listed (maybe a couple more too), so I wouldn't worry about
validation on your end (for the time being).

**Comment by FlanFlanagan** [comment]
Sounds good!

On Wed, May 20, 2015 at 9:19 AM, Robert Carlsen <notifications@github.com>
wrote:

> Radio - I think a text box is fine, and just let the user put it in
> whatever format they want. The cyclus input file can handle all of the
> formats I listed (maybe a couple more too), so I wouldn't worry about
> validation on your end (for the time being).
>
> On Wed, May 20, 2015 at 7:51 AM, mbmcgarry <notifications@github.com>
> wrote:
>
> > I vote for one of the letter-number combos
> >
> >
> > On May 19, 2015, at 10:00 PM, FlanFlanagan <notifications@github.com
> > <mailto:notifications@github.com>> wrote:
> >
> > Hey, pigging backing on this. What would you like nuclide to show in the
> > ui. I don't have behavior for it now
> > On May 19, 2015 9:48 AM, "Robert Carlsen" <notifications@github.com
> > <mailto:notifications@github.com>> wrote:
> >
> > > I don't think so - we should probably add one:
> > >
> > > 92235, 922350000, u235, U235, u-235, U-235
> > >
> > > On Tue, May 19, 2015 at 8:43 AM, Matthew Gidden <
> > notifications@github.com<mailto:notifications@github.com>>
> > > wrote:
> > >
> > > > Merged #146 <https://github.com/cyclus/cyclus.github.com/pull/146>.
> > > >
> > > > —
> > > > Reply to this email directly or view it on GitHub
> > > > <
> https://github.com/cyclus/cyclus.github.com/pull/146#event-308530384>.
> >
> > > >
> > >
> > > —
> > > Reply to this email directly or view it on GitHub
> > > <
> >
> https://github.com/cyclus/cyclus.github.com/pull/146#issuecomment-103532437
> >
> >
> > > .
> > >
> >
> > —
> > Reply to this email directly or view it on GitHub<
> >
> https://github.com/cyclus/cyclus.github.com/pull/146#issuecomment-103736114
> >.
> >
> >
> > *************************************************************************
> > Meghan McGarry, PhD ~ http://mbmcgarry.github.io/
> > Postdoctoral Fellow, Consortium for Verification Technology
> > Computational Nuclear Engineering ~ http://cnerg.engr.wisc.edu
> > Engineering Physics ~ UW-Madison ~ 608-263-6974
> > *************************************************************************
> >
> >
> >
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <
> https://github.com/cyclus/cyclus.github.com/pull/146#issuecomment-103894894
> >
>
> > .
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cyclus/cyclus.github.com/pull/146#issuecomment-103903301>
> .
>



-- 
Robert Flanagan
Graduate Research Assistant
Nuclear and Radiation Engineering Program
Department of Mechanical Engineering
The University of Texas at Austin

**Pull request merged**